### DISCUSSION TITLE
How to apply to GitHub developer student pack?

### DISCUSSION THREAD

**Discussion by Rajath-Bhyrav-S** [community/community#135546] [GitHub Education]
### Hi everyone,

I have been trying to apply to the student developer pack as a student. But the join GitHub education page directs me to the pricing page which does not have an option to apply for the development pack. Any help would be appreciated.

**Comment by davevad93** [comment]
Hi @Rajath-Bhyrav-S ,

You can apply for benefits [here](https://education.github.com/discount_requests/application), although you aren't the first to have that issue, as you can see in these discussions: #131668 , #135388

If the link I provided still redirects you to the pricing page, try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.
